scss/appNav.module.scss

```diff
@@ -79,6 +79,18 @@
   }
 }
 
+.btn-hoverFix {
+  &:hover,
+  &:focus {
+    background-color: color.change(
+      variables.$primary,
+      $lightness: 90%
+    ) !important;
```

Comment:
is it possible to not use `!important`? Its best to be avoided because it breaks natural cascading

Reply:
I've tried using an `id` , using multiple classes as a selector to add more specificity. I wanted to try inline style, but saw that was not recommended either, also not sure if it's possible to add pseudoclasses with inline. 

example of what i tried with id and adding specificity:

appNav.tsx:
`
<div className={`${styles['nav-buttons']}`} id="btnOverride">
    <NavLink
      path={LOGIN_PATH}
      className={`btn m-2 me-lg-3 ${styles['login-btn']} ${styles['btn-hoverFix']}`}
    >
      Login
    </NavLink>
    <NavLink
      path={SIGNUP_PATH}
      className={`btn m-2 me-lg-3 ${styles['signup-btn']} ${styles['btn-hoverFix']}`}
    >
      Signup
    </NavLink>
  </div>
`
appNav.module.scss file:
`#btnOverride .nav-buttons .btn-hoverFix{
  &:hover,
  &:focus {
    background-color: color.change(
      variables.$primary,
      $lightness: 90%
    );
    color: variables.$primary;
    --bs-btn-hover-color: variables.$primary;
    border: 1px solid variables.$primary;
  }
}
`